nrf_rpc/doc/architecture.rst

```diff
@@ -89,11 +89,11 @@ Error handling
 Two kinds of errors might occur when using this library.
 
  Error during parsing of an incoming packet
-    This kind of error cannot be directly returned to the user, e.g. as a return value.
+    These kinds of error cannot be directly returned to the user as a return value.
```

Comment:
Usunąłbym `e.g.`, bo tu tylko wchodzi w grę `return value`, więc to nie jest przykład, bo innej możliwości nie przewiduję.